01_mpileups/samtools_mpileup/samtools_mpileup.cwl

```diff
@@ -0,0 +1,28 @@
+cwlVersion: v1.2
```

Comment:
Same as above in terms of adding a hint for a DockerRequirement. Luckily, you can reuse simply this image already available:
```cwl
 hints:
    DockerRequirement:
      dockerPull: dukegcb/samtools:1.3
```